### PR TITLE
chore(release): update package versions

### PR DESCRIPTION
**Pull request by jnm2377**
Releasing a patch for the following packages:

```
Changes:
 - @carbon/colors: 10.4.0 => 10.4.1
 - carbon-components: 10.6.1 => 10.6.2
 - @carbon/elements: 10.6.0 => 10.6.1
 - carbon-components-react: 7.6.1 => 7.6.2
 - @carbon/sketch: 10.6.0 => 10.6.1 (private)
 - @carbon/test-utils: 10.3.0 => 10.3.1 (private)
 - @carbon/themes: 10.6.0 => 10.6.1
 - @carbon/type: 10.5.0 => 10.5.1
```

Accurate changes to compare => https://github.com/carbon-design-system/carbon/compare/v10.6.1...jnm2377:chore/release-10.6.2

### CODE REVIEW

File: packages/components/docs/sass.md [hunk 1]
```diff
@@ -3665,6 +3673,7 @@ Define theme variables from a map of tokens
   $text-02: map-get($theme, 'text-02') !global;
   $text-03: map-get($theme, 'text-03') !global;
   $text-04: map-get($theme, 'text-04') !global;
+  $text-05: map-get($theme, 'text-05') !global;
```
Comment (jnm2377) [inline packages/components/docs/sass.md:3676]: Looks like ci is failing bc there's a reference to `$text-05` (auto generated in this sass doc) which is a new token. This is coming from of the generated `chore(project): sync generated files [skip ci]` commits.

Comment (jnm2377) [reply packages/components/docs/sass.md:3676]: Should these be excluded from the PR? @joshblack

Comment (jnm2377) [reply packages/components/docs/sass.md:3676]: commit: https://github.com/carbon-design-system/carbon/pull/4006/commits/174d7782277f46f3d83579e84a3314cac7489a3b

**Review by joshblack — APPROVED** [review]

File: packages/components/docs/sass.md (hunk 1 above)
Comment (joshblack) [reply packages/components/docs/sass.md:3676]: It seems like `text-05` was added in: 36113ce679f5e851efe78292298b8f1acd814629 and synced in: https://github.com/carbon-design-system/carbon/commit/99f12f98e226b2ab1872b7379167d827e4fe7bd5

Does that match what you've found?

Comment (jnm2377) [reply packages/components/docs/sass.md:3676]: yes

**Review by designertyler — APPROVED** [review]

**Review by abbeyhrt — APPROVED** [review]

**Pull request closed without merging**